Source/Scene/Primitive.js

```diff
@@ -1217,6 +1218,12 @@ define([
             modelMatrix = this.modelMatrix;
         }
 
+        //>>includeStart('debug', pragmas.debug);
```

Comment:
Why not simplify this to:
```javascript
//>>includeStart('debug', pragmas.debug);
if (!Matrix4.equals(this.modelMatrix, Matrix4.IDENTITY)) {
    throw new DeveloperError('Primitive.modelMatrix is only supported in 3D mode and only for single geometry instances.');
}
//>>includeEnd('debug');
```
and then move it into the `if` block above?  It will be faster and cleaner.